src/renderer/index.js

```diff
@@ -18,6 +18,7 @@ import app from './app';
 import analytics from './analytics';
 
 const { autoUpdater } = remote.require('electron-updater');
+const specialURL = 'lbry://?';
```

Comment:
I think we should call this something else. Maybe `appPageUrl` or something. Just to give some indication of what it is

Reply:
good call.